modules/solid_mechanics/src/materials/ConstitutiveModel.C

```diff
@@ -31,13 +31,13 @@ InputParameters validParams<ConstitutiveModel>()
 }
 
 
-ConstitutiveModel::ConstitutiveModel( const InputParameters & parameters)
+ConstitutiveModel::ConstitutiveModel(const InputParameters & parameters)
   :Material(parameters),
    _has_temp(isCoupled("temp")),
    _temperature(_has_temp ? coupledValue("temp") : _zero),
    _temperature_old(_has_temp ? coupledValueOld("temp") : _zero),
    _alpha(parameters.isParamValid("thermal_expansion") ? getParam<Real>("thermal_expansion") : 0.),
-   _alpha_function( parameters.isParamValid("thermal_expansion_function") ? &getFunction("thermal_expansion_function") : NULL),
+   _alpha_function(parameters.isParamValid("thermal_expansion_function") ? &getFunction("thermal_expansion_function") : NULL),
    _has_stress_free_temp(isParamValid("stress_free_temperature")),
    _stress_free_temp(_has_stress_free_temp ? getParam<Real>("stress_free_temperature") : 0.0),
```

Comment:
Daniel,  This is the constitutive model in solid mechanics, and I've included it in my commit here because I changed some of the spacing, ect, but I am not comfortable with changing the logic in Constitutive model from Solid Mechanics.  If @jasondhales does not like the spacing changes that I've made, I'll just pull this file from the commit.